backend/app/Repositories/EloquentTableVisitorsRepository.php

```diff
@@ -11,9 +11,16 @@
 
 final class EloquentTableVisitorsRepository implements TableVisitorsRepository
 {
-    public function groupByCity(int $website_id, string $from, string $to): Collection
-    {
+    public function groupByCity(
```

Comment:
Create another repository. New visitors and visitors are different entities. And moreover you are passing flag as parameter

Reply:
Fixed